app/models/commands/todo_list/create.rb

```diff
@@ -12,5 +13,9 @@ class TodoList::Create
         metadata: metadata
       )
     end
+
+    def unique_list_name
+      errors.add(:base, 'List name must be unique') unless ::TodoList.where(name: name).empty?
```

Comment:
Because of the structure of the kickstarter app the leading `::` is required to specify we want the Aggregate from the upper scope. Otherwise this refers (I assume) to `Commands::TodoList`. I find it ugly.